vision/use-cases/image_segmentation_layers.ipynb

```diff
@@ -0,0 +1,961 @@
+        "    </a>\n",
+        "  </td>\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/workbench/deploy-notebook?download_url=https://raw.githubusercontent.com/GoogleCloudPlatform/generative-ai/main/vision/use-cases/image_segmentation_layers.ipynb\">\n",
+        "      <img src=\"https://lh3.googleusercontent.com/UiNooY4LUgW_oTvpsNhPpQzsstV5W8F7rYgxgGBD85cWJoLmrOzhVs_ksK_vgx40SHs7jCqkTkCk=e14-rj-sc0xffffff-h130-w32\" alt=\"Vertex AI logo\"><br> Open in Vertex AI Workbench\n",
+        "    </a>\n",
+        "  </td>    \n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://github.com/GoogleCloudPlatform/generative-ai/blob/main/vision/use-cases/image_segmentation_layers.ipynb\">\n",
+        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\"><br> View on GitHub\n",
+        "    </a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "84f0f73a0f76"
+      },
+      "source": [
+        "| | |\n",
+        "|-|-|\n",
+        "|Author(s) | [Katie Nguyen](https://github.com/katiemn) |"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-nLS57E2TO5y"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "### Image Segmentation\n",
+        "\n",
+        "Image Segmentation on Vertex AI brings Google's state of the art segmentation models to developers as a scalable and reliable service.\n",
+        "\n",
+        "With the Vertex AI Image Segmentation API, developers can choose from five different modes to segment images and build AI products.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Gb8FlC1uULsE"
+      },
+      "source": [
+        "In this tutorial, you will learn how to use the Vertex AI API to interact with the Image Segmentation model to:\n",
+        "\n",
+        "- Segment images using different modes\n",
+        "- Turn the masks to layers in a PSD file\n",
+        "- Save the file to a Cloud Storage bucket"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mvKl-BtQTRiQ"
+      },
+      "source": [
+        "## Get Started"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "l2IULNJBtaGS"
+      },
+      "source": [
+        "### Install Vertex AI SDK for Python and Wand"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "PPmd1BWXthqG"
+      },
+      "outputs": [],
+      "source": [
+        "!sudo apt-get install libmagickwand-dev\n",
+        "!pip install Wand\n",
+        "\n",
+        "! pip3 install --upgrade --user google-cloud-aiplatform"
```

Comment:
Can you put the pip installs into one command? and use `%pip` instead of `!pip`?